sagemaker_neo_compilation_jobs/xgboost_customer_churn/xgboost_customer_churn_neo.ipynb

```diff
@@ -615,7 +618,7 @@
     "                                   input_shape={'data':[1, 69]},\n",
     "                                   role=role,\n",
     "                                   framework='xgboost',\n",
-    "                                   framework_version='0.7',\n",
+    "                                   framework_version='latest',\n",
```

Comment:
I recommend against using latest in case this changes.

Reply:
Changed to 0.9, which is the current latest - and verified it works.